deploy/charts/cert-manager/values.yaml

```
@@ -516,48 +516,48 @@ prometheus:
+  # The value must be between 1 and 30 seconds. For more information, see
+  # [Validating webhook configuration v1](https://kubernetes.io/docs/reference/kubernetes-api/extend-resources/validating-webhook-configuration-v1/).
   #
-  # We set the default to the maximum value of 30 seconds. Here's why:
-  # Users sometimes report that the connection between the K8S API server and
+  # The default is set to the maximum value of 30 seconds as
+  # users sometimes report that the connection between the K8S API server and
   # the cert-manager webhook server times out.
   # If *this* timeout is reached, the error message will be "context deadline exceeded",
   # which doesn't help the user diagnose what phase of the HTTPS connection timed out.
   # For example, it could be during DNS resolution, TCP connection, TLS
   # negotiation, HTTP negotiation, or slow HTTP response from the webhook
   # server.
-  # So by setting this timeout to its maximum value the underlying timeout error
+  # By setting this timeout to its maximum value the underlying timeout error
   # message has more chance of being returned to the end user.
   timeoutSeconds: 30
 
-  # Used to configure options for the webhook pod.
-  # This allows setting options that'd usually be provided via flags.
+  # This is used to configure options for the webhook pod.
+  # This allows setting options that would usually be provided using flags.
   # An APIVersion and Kind must be specified in your values.yaml file.
-  # Flags will override options that are set here.
+  # Flags override options that are set here.
   #
   # For example:
   #  apiVersion: webhook.config.cert-manager.io/v1alpha1
   #  kind: WebhookConfiguration
-  #  # The port that the webhook should listen on for requests.
-  #  # In GKE private clusters, by default kubernetes apiservers are allowed to
-  #  # talk to the cluster nodes only on 443 and 10250. so configuring
-  #  # securePort: 10250, will work out of the box without needing to add firewall
+  #  # The port that the webhook listens on for requests.
+  #  # In GKE private clusters, by default Kubernetes apiservers are allowed to
+  #  # talk to the cluster nodes only on 443 and 10250. Configuring
+  #  # securePort: 10250 therefore will work out-of-the-box without needing to add firewall
   #  # rules or requiring NET_BIND_SERVICE capabilities to bind port numbers < 1000.
-  #  # This should be uncommented and set as a default by the chart once we graduate
-  #  # the apiVersion of WebhookConfiguration past v1alpha1.
+  #  # This should be uncommented and set as a default by the chart once
+  #  # the apiVersion of WebhookConfiguration graduates beyond v1alpha1.
   #  securePort: 10250
   config: {}
 
-  # Deployment update strategy for the cert-manager webhook deployment.
-  # See https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+  # The eployment update strategy for the cert-manager webhook deployment.
```

suggestion: typo here, but I think we're echoing with `deployment` and we could remove the word!

```suggestion
  # The update strategy for the cert-manager webhook deployment.
```